src/sagemaker/jumpstart/types.py

```diff
@@ -730,6 +730,7 @@ class JumpStartModelSpecs(JumpStartDataHolderType):
         "training_dependencies",
         "training_vulnerabilities",
         "deprecated",
+        "info_message",
```

Comment:
Any way to use a more specific attribute name which makes it clear immediately to the reader what purpose it serves and when it gets displayed?

Reply:
sure, any suggestions?